### ISSUE TITLE
Duplicate <style> elements for icons and tooltip are constantly added to the <header> when navigating between pages.

### ISSUE THREAD

**Issue by mking200** [vmware/clarity#3077]
## Describe the bug

When you navigate between pages, these 2 `<style>` elements for icons and tooltip below are constantly being generated.  

So the more pages you navigate to, the more copies of these elements will end up on the `<header>`

```
<style type="text/css">
.BDTLL_icon_ok,
.BDTLL_icon_critical,
.BDTLL_icon_alert,
.BDTLL_icon_disabled,
.BDTLL_icon_whitelisted {
    background-color: #252628;
    background-repeat: no-repeat;
    background-size: 76px 76px;
    background-position: 24px 0px;
    margin-top: 10px;
    margin-bottom: 10px;
    height: 76px;
    padding-left: 110px;
    display: inline-block;
}

.BDTLL_icon_ok {
    background-image: url(data:image/png;base64, ... );
}

.BDTLL_icon_critical {
    background-image: url(data:image/png;base64, ... );  
}

.BDTLL_icon_alert {
    background-image: url(data:image/png;base64, ... );
}

.BDTLL_icon_disabled {
    background-image: url(data:image/png;base64, ... );
}

.BDTLL_icon_whitelisted {
    background-image: url(data:image/png;base64, ... );
}

.BDTLL_title {
    color: white;
    font-size: 22px;
    line-height: 30px;
    padding-top: 2px;
}

.BDTLL_description {
    color: #aaaaab;
    font-size: 14px;
    margin-right: 12px;
}
</style>

<style type="text/css">
.BDTLL_tooltip {
    background-color: #252628;
    width: 540px;
    border-radius: 5px;
    display: block;
    position: absolute;
    z-index: 100;
    overflow: hidden;
    margin-left: -20px;
}

.BDTLL_status {
    cursor: pointer;
    display: inline;
    margin-right: 3px;
    width: 16px;
    height: 16px;
}
</style>
````

## How to reproduce

This behavior is also present in the [Clarity Demo App](https://clarity-ks.surge.sh)

1.  Open the inspect element of the browser.  And view the end of `<header>` element.
2.  Navigate to any of the same 2 pages. Example: "badges" and "cards"
3.  Go back and forth between these 2 pages.
4.  You will notice the same `<style>` elements are being added each time you switch pages.

## Expected behavior

Only 1 copy of these `<style>` element should be generated.

**Comment by gnomeontherun** [comment]
I'm not seeing this behavior on the test site that you pointed to, and this behavior is normally controlled by Angular to inject CSS into the page based on the active component. Can you provide a reproduction of your own scenario that demonstrates this?

**Comment by mking200** [comment]
@gnomeontherun here is a screen recording of the bug:
https://www.dropbox.com/s/qe83kcxyt0ymm5q/clarity_screencast.gif?raw=1

The same behavior also happens on my local project with these versions of Angular and Clarity:

Angular: 7.2.2
Clarity: 1.0.4

**Comment by mking200** [comment]
@youdz sorry, it is not from Clarity.  This also happens when I visit other websites, so it must be from a browser plugin.  Closing this issue now.

**Comment by drhax9908** [comment]
sorry to post on this very old closed issue, but I had a similar issue with my own website, and I found the source of that extension altered code. If you use BitDefender TrafficLight, it will add code to the site, the div tag with id BDTLL_feedback is one of the things it adds.. No clue why it was adding style code however. I confirmed it was BitDefender's extension by turning it off. I honestly thought I had been compromised for a while. Good luck with coding
